monkey/monkey_island/cc/services/config.py

```diff
@@ -251,7 +280,11 @@ def reset_config():
         PostBreachFilesService.remove_PBA_files()
         config = ConfigService.get_default_config(True)
         ConfigService.set_server_ips_in_config(config)
-        ConfigService.update_config(config, should_encrypt=False)
+        try:
+            mode = get_mode()
+            ConfigService.update_config_per_mode(mode, config, should_encrypt=False)
+        except ModeNotSetError:
```

Comment:
When would this happen? If the mode is not set, user can't go to configuration. Maybe we shouldn't catch the exception then?

Reply:
When you start over. If I don't catch it, it throws the exception and doesn't update the config.